src/dune_engine/action.ml

```diff
@@ -319,7 +319,6 @@ let is_useful_to distribute memoize =
     | false -> Clearly_not
 ;;
 
-let is_useful_to_sandbox = is_useful_to false false
```

Comment:
Looks like the lines below can now be simplified (the first argument can just be dropped).

Reply:
done here https://github.com/ocaml/dune/pull/9058